src/components/StaticHeaderPageLayout.js

```diff
@@ -0,0 +1,86 @@
+import _ from 'underscore';
+import React from 'react';
+import PropTypes from 'prop-types';
+import {ScrollView, View} from 'react-native';
+import headerWithBackButtonPropTypes from './HeaderWithBackButton/headerWithBackButtonPropTypes';
+import HeaderWithBackButton from './HeaderWithBackButton';
+import ScreenWrapper from './ScreenWrapper';
+import styles from '../styles/styles';
+import themeColors from '../styles/themes/default';
+import * as StyleUtils from '../styles/StyleUtils';
+import useWindowDimensions from '../hooks/useWindowDimensions';
+import FixedFooter from './FixedFooter';
+
+const propTypes = {
+    ...headerWithBackButtonPropTypes,
+
+    /** Children to display in the lower half of the page (below the header section w/ an animation) */
+    children: PropTypes.node.isRequired,
+
+    /** The illustration to display in the header. Can be either an SVG component or a JSON object representing a Lottie animation. */
+    // illustration: PropTypes.oneOfType([PropTypes.func, PropTypes.object]).isRequired,
+
+    /** The background color to apply in the upper half of the screen. */
+    backgroundColor: PropTypes.string,
+
+    /** A fixed footer to display at the bottom of the page. */
+    footer: PropTypes.node,
+};
+
+const defaultProps = {
+    backgroundColor: themeColors.appBG,
+    footer: null,
+};
+
+function StaticHeaderPageLayout({backgroundColor, children, image: Image, footer, imageContainerStyle, style, ...propsToPassToHeader}) {
+    const {windowHeight} = useWindowDimensions();
+    return (
+        <ScreenWrapper
+            style={[StyleUtils.getBackgroundColorStyle(backgroundColor)]}
+            shouldEnablePickerAvoiding={false}
+            includeSafeAreaPaddingBottom={false}
+            offlineIndicatorStyle={[StyleUtils.getBackgroundColorStyle(themeColors.appBG)]}
+        >
+            {({safeAreaPaddingBottomStyle}) => (
+                <>
+                    <HeaderWithBackButton
+                        // eslint-disable-next-line react/jsx-props-no-spreading
+                        {...propsToPassToHeader}
+                        titleColor={backgroundColor === themeColors.appBG ? undefined : themeColors.textColorfulBackground}
```

Comment:
i think we can define a variable and put this backgroundColor === themeColors.appBG ? undefined : themeColors.textColorfulBackground - in case it's duplicating multiple times

Reply:
the first one ending with `textColorfulBackground` and the second with `iconColorfulBackground` so they are different
I moved it out of the render